File: c#/ShortLongShort.cs
// input 2 strings and return short+long+short with shorter strings on outside

public class ShortLongShort
{
	public static string Solution(string a, string b)
	{
		return a.Length > b.Length ? b + a + b : a + b + a;
	}

	public static void Main(string[] args)
	{
		System.Console.WriteLine(Solution("1", "22")); // 1221
		System.Console.WriteLine(Solution("22", "1")); // 1221
	}
}

File: java/shortLongShort.java
// output a string with short+long+short

public class shortLongShort {
  public static String solution(String a, String b) {
    String shortStr = a;
    String longStr = b;
    if (a.length() > b.length()) {
      shortStr = b;
      longStr = a;
    }
    return shortStr + longStr + shortStr;
  }

  public static void main(String[] args) {
    System.out.println(solution("11", "2")); // 2112
    System.out.println(solution("2", "11")); // 2112
  }
}